cmd/app.go

```diff
@@ -658,40 +673,45 @@ func templateInstall(ctx *cli.Context) error {
 		app.TargetNamespace = namespace
 	}
 
+	app.Wait = ctx.Bool("wait")
```

Comment:
should wait boolflag have a default of true? false? what happens if you set app.Wait to nil or whatever comes through from the not setting BoolFlag Value?

Reply:
wait a regular bool with default false, which is intended. I tested without using wait.

Reply:
then we're good